quarkus/admin/src/main/java/org/apache/polaris/admintool/BootstrapCommand.java

```diff
@@ -38,20 +39,37 @@ public class BootstrapCommand extends BaseCommand {
   List<String> realms;
 
   @CommandLine.Option(
-      names = {"-c", "--credential"},
-      paramLabel = "<realm,clientId,clientSecret>",
```

Comment:
while I agree that supporting well-formed JSON is valuable, I believe the value is realized mostly in automated use case. This old option, however, is easier to use for humans, IMHO... Why not keep both options available for users to choose what it convenient for them?

For example: `--json` for JSON, and `--client-id`, `--client-secret` for plain strings?

Reply:
> This old option, however, is easier to use for humans, IMHO...

I really do not think that is the case for the current format.

As for supporting both the old and the new format (and maybe field-specific args like you suggest), I think we're best off just having one way to do the same thing here.

Reply:
I did not mean the old "format", but the old approach of using plain text for ID/secret :) That's what I suggest: add new options (`--client-id`, `--client-secret`) to allow the old _way_ of using plain text values to bootstrap.

Reply:
I'm suggesting to delegate the choice of which way to go to the user.

Option 1:
`bootstrap --realm r1 --client-id c1 --client-secret s1`
[... do something else ...]
`bootstrap --realm r2 --client-id c2 --client-secret s2`
etc....

Option 2:
`bootstrap --json '{...}'` - here all realm IDs and client ID/secret pairs are in the JSON.

Reply:
Just to clarify: Option 1 only bootstraps one realm at a time, correct? I think I can live with that.

Reply:
FYI, I'm working on #878... I'm leaning towards a file in the following syntax:

```json
{
  "credentials" : {
    "realm1" : {
      "client-id" : "client1",
      "client-secret" : "secret1"
    },
    "realm2" : {
      "client-id": "client2",
      "client-secret": "secret2"
    }
  }
}
```

YAML will also be supported:

```yaml
credentials:
  realm1:
    client-id: client1
    client-secret: secret1
  realm2:
    client-id: client2
    client-secret: secret2
```

Reply:
One last comment, sorry... I think the admin tool should require the user to enter credentials for each realm to bootstrap. In real life, it makes little sense to bootstrap with random credentials. Wdyt? At the least the file syntax for now requires the credentials to be specified.

Reply:
`bootstrap --json` feels a little unclear to me; e.g. should the `--print-credentials` option also go inside that JSON? It seems to imply that we are bootstrapping in JSON "mode" or providing all options into a JSON. While in fact, the JSON is just a set of `credentials`.

Since we call it `credentials`, I think having a second way to do the exact same thing could be confusing, i.e. having both a `credentials` option and a `client-id` option.

Reply:
`--print-credentials` is just a CLI option. I do no think it should be inside JSON.

As for `--json` being unclean, how about:

Option 1: `bootstrap --realm ... [ --client-id ... --client-secret ...]` (as above)
Option 2: `bootstrap --realm-json <json>`, where `<json>` is from from @adutra 's example
Option 3: `bootstrap --realm-yaml ...` - same as JSON, but in YAML format